packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -73,6 +73,29 @@ const logAndFormatError = (err) => {
     }
 }
 
+// Because multi-value params are not supported in `aws-serverless-express` we need to re-add
+// the stripped duplicate params.
+const FIX_LOCATION_SEARCH = (req, location) => {
```

Comment:
ALL CAP? 🤨

Reply:
Lol 😆 same as @alexvuong ... Caps is for constants... I would name it `fixMePleaseLocationSearch`

Reply:
I did this intentionally to make it super visible. Doing this will ensure that we remove it once we upgrade the underlying library.